README.md

```diff
@@ -69,13 +72,28 @@ Then, follow these steps for creating a contribution:
 
 This add-on is distributed under Apache License 2.0. For license terms, see LICENSE.txt.
 
-TEMPLATE_ADDON is written by Flowing Code S.A.
+CronExpressionEditor is written by Flowing Code S.A.
```

Comment:
Name should be "Cron Expression Editor Add-on" as explained [here](https://github.com/FlowingCode/TechnicalDocs/tree/master/addons#naming-convention).

Reply:
Please check that fix is incomplete. Name should be "**Cron Expression Editor Add-on**" with space between words as specified in the documentation shared in the previous comment.